src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -62,23 +62,22 @@ ucs_config_field_t uct_rc_mlx5_common_config_table[] = {
    ucs_offsetof(uct_rc_mlx5_iface_common_config_t, exp_backoff),
    UCS_CONFIG_TYPE_UINT},
 
-  {"CYCLIC_SRQ_ENABLE", "try",
-   "Enable using the \"cyclic\" SRQ type (SRQ is organized as a continuous \n"
-   "array of WQEs), otherwise - using the \"list\" SRQ type (SRQ is organized \n"
-   "as a buffer containing linked list of WQEs.",
-   ucs_offsetof(uct_rc_mlx5_iface_common_config_t, cyclic_srq_enable),
-   UCS_CONFIG_TYPE_TERNARY},
-
-  {"LINKED_LIST", "n",
-   "TBD",
-   ucs_offsetof(uct_rc_mlx5_iface_common_config_t, linked_list_srq),
-   UCS_CONFIG_TYPE_BOOL},
+  {"SRQ_TOPO", "cyclic,cyclic_emulated",
+   "List of SRQ topology types in order of preference. Supported types are:\n"
+   "\n"
+   "list              SRQ is organized as a buffer containing linked list of WQEs.\n"
+   "\n"
+   "cyclic            SRQ is organized as a continuos array of WQEs. Requires DEVX.\n"
+   "\n"
+   "cyclic_emulated   SRQ is organized as a continuos array of WQEs. Doesn`t require DEVX.",
```

Comment:
SRQ is organized as a continuous array of WQEs, but HW treats it as a linked list. Doesn`t require DEVX.